### PR TITLE
Fix bug where update_many would crash if result was empty

### PR DESCRIPTION
**Pull request by nmenardg**
https://www.pivotaltracker.com/n/projects/1278176/stories/130012505

### CODE REVIEW

File: ramses/views.py [hunk 1]
```diff
@@ -331,9 +331,12 @@ def update_many(self, **kwargs):
         by ES in the 'index' method (so user updates what he saw).
         """
         db_objects = self.get_dbcollection_with_es(**kwargs)
-        items = db_objects.all()
+
+        if not isinstance(db_objects, list):
```
Comment (nikitagromov) [inline ramses/views.py:335]: I think we don't need this check, `self.Model._update_many` can handle list and query, so you can just pass db_objects to _update_many function

**Comment by nikitagromov** [comment]
Approved

**Comment by nmenardg** [comment]
I have a test that's not passing...

**Comment by nmenardg** [comment]
@nikitagromov `_update_many` does accept a query, but it changes it in ways that make relationships.robot tests break. I think it's because these tests run an update_many with `id=(id OR id)` parameter.

So I'm keeping the `.all()` call

**Comment by nikitagromov** [comment]
@nmenardg Ok, np

**Pull request merged**